### PR TITLE
Add P2A certified-connector/Converter

### PR DESCRIPTION
**Pull request by stsiek**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2461,6 +2461,315 @@
           }
         ]
       }
+    },
```
Comment (v-eisljami) [inline certified-connectors/PDF4me Connect/apiDefinition.swagger.json:2464]: PR should contain only files for one connector. Please either exclude this file from the PR or submit new PR with only files related to the Converter connector.

File: certified-connectors/Converter/Readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (v-eisljami) [inline certified-connectors/Converter/Readme.md:1]: Readme file is not in the correct format. Please refer to this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for readme template.

File: certified-connectors/Converter/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/ConvertCsvdataToExcel": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (v-eisljami) [inline certified-connectors/Converter/apiDefinition.swagger.json:38]: Empty schemas are not allowed in the swagger definition.

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 1 above)
Comment (Amjed-Ayoub) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:2464]: Hi @stsiek, I hope you are doing well. it looks like you are updating "PDF4me Connect" as well in this PR. I suggest you close this PR and resubmit a new PR with the appropriate connector files only.
Please let us know if you have any questions.
Thank you very much

**Comment by Amjed-Ayoub** [comment]
Hi @stsiek, I hope you are doing well. I am following up with you to check on how updating your pull request is going.
Please do not hesitate to reach out to me if you have any questions.
Thank you very much

**Comment by stsiek** [comment]
I have changed the open issues

File: certified-connectors/Converter/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "CSV-Data",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important",
+              "title": ""
```
Comment (sriyen-msft) [inline certified-connectors/Converter/apiDefinition.swagger.json:62]: @stsiek Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Converter/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+        "description": "Convert CSV-Data to JSON",
+        "parameters": [
+          {
+            "name": "CSV-Data",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important",
+              "title": ""
```
Comment (sriyen-msft) [inline certified-connectors/Converter/apiDefinition.swagger.json:98]: @stsiek Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Converter/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+            "x-ms-summary": "SP-Result"
+          },
+          {
+            "name": "JSON",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important",
+              "title": ""
```
Comment (sriyen-msft) [inline certified-connectors/Converter/apiDefinition.swagger.json:180]: @stsiek Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Converter/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+          "tooltip": "Stellen Sie \"Api-Key (free on \"https://power2apps.de/apikey\")\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```
Comment (sriyen-msft) [inline certified-connectors/Converter/apiProperties.json:18]: @stsiek Please refer the template provided for apiProperties.json file and update it to the default iconBrandColor.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json

File: certified-connectors/Converter/apiProperties.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    "capabilities": [],
+    "publisher": "Siek, Stephan"
```
Comment (sriyen-msft) [inline certified-connectors/Converter/apiProperties.json:20]: @stsiek Please add the stackOwner property.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Converter/apiProperties.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+          "tooltip": "Stellen Sie \"Api-Key (free on \"https://power2apps.de/apikey\")\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
```
Comment (sriyen-msft) [inline certified-connectors/Converter/apiProperties.json:18]: @stsiek This is the default iconBrandColor for all the independent publisher connectors, Please make sure it is the background color of your connector icon.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@stsiek This is the default iconBrandColor for all the independent publisher connectors, Please make sure it is the background color of your connector icon.

**Pull request merged**